<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>678</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1020</width>
    <height>678</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1020</width>
    <height>678</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(75, 121, 161, 255), stop:1 rgba(40, 62, 81, 255));}
*{font-family:  century gothic;}
QLineEdit{
background: transparent;
border: none;
color: #717072;
border-bottom: 1px solid #717072;
font-size: 24pt;

}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>110</y>
     <width>660</width>
     <height>461</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{
background: rgba(51, 51, 51, 0.8);
border-radius: 15px;
}</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>367</width>
      <height>414</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font-family:  century gothic;
font-size: 40pt;
color: white;</string>
       </property>
       <property name="text">
        <string>Введите тему:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>100</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="theme_input">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Введите тему</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>120</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">
#pushButton{

border-radius: 15px;
background-color: #fff;
font-size: 20pt;}


#pushButton:hover{
transition: transform 1s;


border: 5px, 5px, 5px, 5px solid #717072;
background: #eee;
font-size: 20pt;}


</string>
       </property>
       <property name="text">
        <string>Далее</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>590</y>
     <width>609</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar::chunk
{
border-radius:11px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:1,y2:0,stop:0 #01FAFF,stop:1  #26B4FF);
}
#progressBar{
height:22px;
Text-align: center;
font-size:14px;
color:white;
border-radius:11px;
background: #1D5573 ;
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="menu">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#menu:hover{
border-radius: 10px;
background: rgba(255, 255, 255, 0.5);
}
#menu{
background: transparent;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/squares.png</normaloff>icons/squares.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
